backend/api/controllers/index.js

```diff
@@ -151,10 +137,60 @@ const resubscribe = async (request, response) => {
 	}
 };
 
+const DeletionErrors = {
+	INVALID_TOKEN: DeletionError('User token is invalid'),
+	NONEXISTENT_USER_TO_DELETE: DeletionError('The user that you are trying to delete does not exist'),
+	NONEXISTENT_USER_TO_RECOVER: DeletionError('The user that you are trying to recover does not exist')
+};
+
+const destroy = async (request, response) => {
+	try {
+		const user = await User.findByRequest(request);
+		if (user) {
+			user.userScheduler = new UserScheduler();
```

Comment:
Make this the default scheduler -- if scheduler is undefined, have the user model instantiate the `UserScheduler`.

Reply:
Will implement this improvement at a later stage.